test/amperity/gocd/secret/vault/plugin_test.clj

```diff
@@ -0,0 +1,214 @@
+(ns amperity.gocd.secret.vault.plugin-test
```

Comment:
was this just renamed from `plugin_api.clj`? having some deja vu here.

Reply:
This was renamed but I added more tests too 👍

Reply:
Ok thanks. Its good practice to make separate commits of 1) moving/renaming 2) then changes to the new location so git can follow better and know it was a mv not a rm.

Reply:
eh, ignore that you did what i suggested.

https://github.com/amperity/gocd-vault-secrets/commit/08db21cf076b5539552f98cb1f2132cd834b1a3c#diff-7ccde7940a09d984ad2a39bbd832dff7

Must be how github is rendering this then.

Reply:
Yeah, I was surprised it wasn't tracked as a rename. I woulda done it in its own PR if I knew it would do a delete and new file. Will do refactors like that as their own PR in the future just in case